iidm-xml-converter/src/main/java/eu/itesla_project/iidm/xml/ConnectableXml.java

```diff
@@ -128,11 +128,15 @@ public static void readCurrentLimits(Integer index, Supplier<CurrentLimitsAdder>
         adder.setPermanentLimit(permanentLimit);
         XmlUtil.readUntilEndElement("currentLimits" + indexToString(index), reader, () -> {
             if ("temporaryLimit".equals(reader.getLocalName())) {
-                int acceptableDuration = Integer.valueOf(reader.getAttributeValue(null, "acceptableDuration"));
-                float limit = Float.valueOf(XmlUtil.readUntilEndElement("temporaryLimit", reader, null));
+                String name = reader.getAttributeValue(null, "name");
+                Integer acceptableDuration = XmlUtil.readOptionalIntegerAttribute(reader, "acceptableDuration");
+                float value = XmlUtil.readOptionalFloatAttribute(reader, "value");
```

Comment:
Could be replaced by float value = XmlUtil.readOptionalFloatAttribute(reader, "value", Float.MAX_VALUE); ?
Maybe not due to "Float.isNaN(value)" check below.